src/ansys/aedt/core/visualization/post/common.py

```diff
@@ -1350,6 +1350,9 @@ def create_report(
             - For a far fields plot, specify the name of an infinite sphere.
             - For Maxwell 2D/3D Eddy Current solution types this can be provided as a dictionary
             where the key is the matrix name and value the reduced matrix.
+            - For Circuit Design, this can provide the plots' time range as a dictionary
+            where the keys are ``"time_start"`` and ``"time_stop"``.
+            By default ``"time_start"`` is 0ps and the ``"time_stop"`` is 10ns.
```

Comment:
@anur7 @nunesanderson are you setting these values somewhere? if you say that the start time for Circuit is 0ps, is this value set somewhere? or do you just say that because it is like that in AEDT?